eng/versioning/version_client.txt

```diff
@@ -122,7 +122,7 @@ com.azure:azure-perf-test-parent;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-search-documents;11.4.12;11.5.0-beta.12
 com.azure:azure-search-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-security-attestation;1.1.3;1.2.0-beta.1
```

Comment:
Is this module not included in eng/jacoco-test-coverage/pom.xml and does not have a respective perf module?

Reply:
It's not, as far as I know.